poetry.lock

```diff
@@ -0,0 +1,17 @@
+# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
```

Comment:
Should this file be managed, or is it generated via the included `.toml` file?

Reply:
> Should this file be managed?

[Yes](https://python-poetry.org/docs/basic-usage/#committing-your-poetrylock-file-to-version-control)

> is it generated via the included .toml file?

No. The lock file is managed and generated by Poetry. The `pyproject.toml` file is a handy way to configure different aspects of your project in a single file. For example, a relation of the project description, authors, documentation URLs, dependencies, optional dependencies or configuration options for tools used by the developers (like [ruff](https://github.com/astral-sh/ruff)